Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -160,22 +164,39 @@ public class CarPlayManager: NSObject {
                                                selector: #selector(navigationCameraStateDidChange(_:)),
                                                name: .navigationCameraStateDidChange,
                                                object: carPlayMapViewController?.navigationMapView.navigationCamera)
+        os_log("CarPlayManager subscribed for notifications.", log: logger, type: .info)
     }
     
     func unsubscribeFromNotifications() {
         NotificationCenter.default.removeObserver(self,
                                                   name: .navigationCameraStateDidChange,
                                                   object: carPlayMapViewController?.navigationMapView.navigationCamera)
+        os_log("CarPlayManager suspended notifications.", log: logger, type: .info)
+    }
+    
+    @available(iOS 15.0, *)
+    func getLogEntries() throws -> [OSLogEntryLog] {
```

Comment:
Is it for debugging? If so, don't forget to remove this

Reply:
This is for retrieving log entries if you want to see them printed to a file instead of looking at the log via the Console. This functionality is only available via iOS 15.

Reply:
I think there are several build errors because of `OSLogEntryLog`. For example getting: error: cannot find type 'OSLogEntryLog' in scope.